### ISSUE TITLE
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult'

### ISSUE THREAD

**Issue by moerwald** [PowerShell/PSReadLine#2659]
Getting below exception if starting PowerShell from Windows Terminal.

### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0-preview.7
OS: Microsoft Windows 10.0.19042
BufferWidth: 209
BufferHeight: 52

Last 0 Keys
```

```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x60004a6+0x0
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x6000478+0x7
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics) in Microsoft.PowerShell.PSReadLine2.dll:token 0x60000f2+0x118
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken) in Microsoft.PowerShell.PSReadLine2.dll:token 0x60000eb+0x60
```

**Comment by Ewerton** [comment]
Same problem here. It started to happen right after I updated PowerShell to the Preview7

**Comment by daxian-dbw** [comment]
This is a known issue and has been addressed in 2.2.0-beta3 release of PSReadLine. Please upgrade to use 2.2.0-beta3 with the latest 7.2.0-preview.9 PowerShell.
